_posts/2024-08-15-n-of-1-fitness-data.md

```diff
@@ -0,0 +1,146 @@
+# N-of-1 Trials and Analyzing Your Own Fitness Data
+
+## Do I really sleep worse after drinking alcohol?
+
+### By [Merete Lutz](https://meretelutz.github.io/)
```

Comment:
Suggested change for the format:

---

layout: post
title: "N-of-1 Trials and Analyzing Your Own Fitness Data"
subtitle: by [Merete Lutz](https://meretelutz.github.io/)

---

## Do I really sleep worse after drinking alcohol?

Reply:
Sounds good! I've made the change.